Core/Object Arts/Dolphin/ActiveX/OCX/AXControlSite.cls

```diff
@@ -903,6 +926,7 @@ usesAmbientColors
 !AXControlSite categoriesFor: #showHatching!ambient properties!public!testing! !
 !AXControlSite categoriesFor: #sink!accessing!public! !
 !AXControlSite categoriesFor: #state!binary filing!private! !
+!AXControlSite categoriesFor: #storeOn:indent:variables:!printing!public! !
```

Comment:
Should I change the categories of the existing #'storeString' and #'storeOn:' methods?